src/app/pages/FrontPage/index.test.jsx

```diff
@@ -17,7 +17,7 @@ const requestContextData = {
 };
 
 const FrontPageWithContext = props => (
-  <ToggleContextProvider>
+  <ToggleContextProvider service="mundo" origin="https://www.test.bbc.com">
```

Comment:
Shall we update the service to match the service provided to the ServiceContextProvider below?